_data/master/navbars/networking.yml

```diff
@@ -20,7 +20,7 @@ toc:
   path: /networking/external-connectivity
 - title: Configuring a node IP address and subnet
   path: /networking/node
-- title: Configuring MTU
+- title: Configure MTU to maximize network performance
```

Comment:
Shouldn't this say "Configure MTU"? (drop the "ing")

Reply:
Fixed/approved